### DISCUSSION TITLE
Anyone interested in a tiny CYD SD Breakout card?

### DISCUSSION THREAD

**Discussion by peteDDD** [witnessmenow/ESP32-Cheap-Yellow-Display#202] [Show and tell]
I just designed a tiny SD breakout card for the CYD.  The ones you can buy online are way to big to put in a small enclosure with the display.  I could put it up on one of the board shops for order if there is enough interest.  (could also be used for other SD card breakout needs but is labeled to match the CYD pinouts.  It will be a few weeks before I get them and test them for myself.

updated rendering:
![Micro SD Sniffer 2 320px](https://github.com/user-attachments/assets/667ec330-969c-4679-a6e0-8a64e26a4463)

See below for zip of files... hopefully someone can put it up in the hardware files.

**Comment by witnessmenow** [comment]
Looks good! Please feel free to add it to the hardware section if you'd like!

**Reply by peteDDD** [reply]
I'd be glad to.  How does one go about doing that?

**Reply by Misiu** [reply]
You must do a PF to the Hardware folder.
BTW can you share the files?

**Reply by peteDDD** [reply]
I have a folder ready but when I to add it, I find that uploads are disabled.   If someone wants to post it, here is a zip.
[Micro_SD_Breakout_Card.zip](https://github.com/user-attachments/files/17097671/Micro_SD_Breakout_Card.zip)

**Reply by Misiu** [reply]
Thank you for the files.
I've downloaded them and tried to create a JLCPCB order. Did you change any settings in their wizard?
![image](https://github.com/user-attachments/assets/924f907e-e125-4a19-88e9-32fe85405a8c)
The dimensions are 21.55 x 11 mm and the thickness is 1.6 mm.
With the standard shipping, it costs $2 for 5 pieces 🚀

**Reply by Misiu** [reply]
> I have a folder ready but when I to add it, I find that uploads are disabled. If someone wants to post it, here is a zip. [Micro_SD_Breakout_Card.zip](https://github.com/user-attachments/files/17097671/Micro_SD_Breakout_Card.zip)

I think you must fork the project, add files to your repo and then create pull request to this repo

**Comment by KevynWithaY** [comment]
yes that's awesome

**Comment by brentmc79** [comment]
@peteDDD did you ever do anything with this? I'm curious if you tested it, or if it's available anywhere...

**Reply by peteDDD** [reply]
I did make a handful of them along with another PCB order.  They work just fine.  If you have another PCB order in the works, it is pennies per board to add these to the order.  It's the shipping that kills you.

**Comment by Misiu** [comment]
So if I understand correctly this will allow us to reuse/reporpuse GPIO5, GPIO18, GPIO19 and GPIO23?

I just found this product: https://www.elecrow.com/microsd-breakout-board.html, maybe it could be used?

**Reply by KevynWithaY** [reply]
> So if I understand correctly this will allow us to reuse/reporpuse GPIO5, GPIO18, GPIO19 and GPIO23?
> 
> I just found this product: https://www.elecrow.com/microsd-breakout-board.html, maybe it could be used?
> 

Nice find

**Reply by Misiu** [reply]
> > So if I understand correctly this will allow us to reuse/reporpuse GPIO5, GPIO18, GPIO19 and GPIO23?
> > I just found this product: https://www.elecrow.com/microsd-breakout-board.html, maybe it could be used?
> 
> Nice find

Thx

So this should allow me to reuse SD card pins to as standard GPIO's? To control relays or as inputs

**Comment by brentmc79** [comment]
I got a few of these from Aliexpress, but I haven’t tried them out yet.

New Micro SD Sniffe Compatible With TF Card Adapter Plate Universal 1pc
https://a.aliexpress.com/_m0BvNHw

**Comment by peteDDD** [comment]
any of the breakout boards are fine.  The one I drew up offers the labeling of the pins relative to the CYD pinout which is handy.

**Comment by peteDDD** [comment]
Yes, 1.6mm thick and default settings.

**Comment by peteDDD** [comment]
Understood... When I gave time...

**Reply by Misiu** [reply]
Done in https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/pull/256
